src/server/services/tips/predicates.test.ts

```diff
@@ -0,0 +1,513 @@
+import MockDate from 'mockdate';
+import { ApiResponse, ApiSuccessResponse } from '../../../universal/helpers';
+import { BRPData, BRPDataFromSource } from '../../../universal/types';
+import BRP from '../../mock-data/json/brp.json';
+import WPI_STADSPAS from '../../mock-data/json/wpi-stadspas.json';
+import WPI_AANVRAGEN from '../../mock-data/json/wpi-aanvragen.json';
+import UITKERINGEN from './mock-data/wpi_aanvragen.mock.json';
+import TONK from './mock-data/tonk.mock.json';
+import TOZO from './mock-data/tozo.mock.json';
+import TOERISTISCHE_VERHUUR from './mock-data/vakantie_verhuur.mock.json';
+import { transformBRPData } from '../brp';
+import { StadspasResponseDataTransformed } from '../wpi/api-service';
+import { WpiRequestProcess } from '../wpi/wpi-types';
+
+import {
+  hasBijstandsuitkering,
+  hasBnBVergunning,
+  hasDutchNationality,
+  hasKidsBetweenAges2And18,
+  hasStadspasGroeneStip,
+  hasToeristicheVerhuurVergunningen,
+  hasTozo,
+  hasValidId,
+  hasValidStadspasRequest,
+  hasVerhuurRegistrations,
+  is18OrOlder,
+  isBetween17and18,
+  isLivingInAmsterdamLessThanNumberOfDays,
+  isReceivingSubsidy,
+  not,
+  or,
+  previouslyLivingInAmsterdam,
+} from './predicates';
+import { CaseType } from '../../../universal/types/vergunningen';
+import {
+  ToeristischeVerhuurRegistratieDetail,
+  ToeristischeVerhuurVergunning,
+} from '../toeristische-verhuur';
+import { TipsPredicateFN } from './tip-types';
+
+describe('predicates', () => {
+  beforeAll(() => {
+    MockDate.set('2022-07-25');

```

Comment:
Om er wat meer nadruk op te leggen kun je de mock date misschien wat duidelijker definiëren.

```suggestion
    const DATE_TO_TEST_AGAINST = '2022-07-25';
    MockDate.set(DATE_TO_TEST_AGAINST);

```